.github/workflows/validate-category-on-pr.yaml

```diff
@@ -0,0 +1,62 @@
+name: Validate Categories on PR
+
+# read-only repo token
+# no access to secrets
+on:
+  pull_request:
```

Comment:
add on `push` event as well. `ghes` branch has direct pushes.

Reply:
ghes branch is updated only through the sync-ghes workflow, I believe. 
Also, we require a PR to comment if needed. Otherwise, the validation script is run, but the result is not shared anywhere.

Reply:
Got it. So, essentially master branch is the gatekeeper for ensuring correct categories.